tests/contrib/django/django_app/settings.py

```diff
@@ -1,8 +1,24 @@
 import os
 
+from ddtrace import tracer
+from ddtrace.filters import TraceFilter
+
+
+class PingFilter(TraceFilter):
+    def process_trace(self, trace):
+        return None if trace and trace[0].trace_id == 1 else trace
```

Comment:
a comment would be nice

because this is saying ignore this trace if the trace id is `1`?